components/WhyMeditate/WhyMeditate.js

```diff
@@ -0,0 +1,59 @@
+import { PauseRounded, PlayArrowRounded } from '@mui/icons-material';
+import React from 'react';
+import { AudioWithSubs } from './AudioWithSubs';
+import styles from './WhyMeditate.module.css';
+import cx from 'classnames';
+
+export function WhyMeditate() {
+  const [pause, setPause] = React.useState(false);
+  const [currentProgress, setCurrentProgress] = React.useState(0);
+  console.log(currentProgress);
+  return (
+    <div className={styles.contentContainer} id="why-meditate">
+      <h1 className={styles.title}>Why Meditate?</h1>
+      <p className={styles.reason}>
+        Alan Watts, the British philosopher, writer, and speaker, makes a great
+        concise case for mediation in this recording.
+      </p>
+      <div
+        className={cx(styles.playerContainer, { [styles.isPlayed]: !pause })}
```

Comment:
`{ [styles.isPlayed]: !pause }` this is a bit of a problem, because the default state is paused. 

See how the default state looks,  even though it's not playing

![image](https://user-images.githubusercontent.com/17054134/150796502-600f5e0f-d65b-48eb-8b46-75c9bda4c093.png)